tests/purs/psci/DefineItFail.purs

```diff
@@ -0,0 +1,2 @@
+-- @prints NoItInREPL
```

Comment:
I'd like to make this directive like the `@shouldFailWith` directive in the compiler tests, but for checking either for a specific failure in the REPL or output text, whichever works. I'm having issues getting there, though, so any help is welcome.

Reply:
Did you get this working?

Reply:
No